core/storage/feedback/gae_models.py

```diff
@@ -89,6 +89,12 @@ def get_deletion_policy():
         """General feedback thread needs to be pseudonymized for the user."""
         return base_models.DELETION_POLICY.LOCALLY_PSEUDONYMIZE
 
+    @staticmethod
+    def get_export_policy():
+        """Model contains user data."""
+        return base_models.EXPORT_POLICY.CONTAINS_USER_DATA
+
+
```

Comment:
Drop one of these newlines. Adjacent methods in a class should only be separated by a single newline.

Ditto below -- this is repeated in several places.

Reply:
@seanlip I'll make these changes, but just as above, I think that this should be a part of the linter.

Reply:
Done